pkg/services/ssosettings/api/api.go

```diff
@@ -99,6 +99,20 @@ func (api *Api) updateProviderSettings(c *contextmodel.ReqContext) response.Resp
 	return response.JSON(http.StatusNoContent, nil)
 }
 
+// swagger:route DELETE /ssoSettings/{key} sso_settings removeProviderSettings
+//
+// # Remove SSO Settings
+//
+// # Remove an SSO Settings entry by Key
+//
+// If you have role based access control enabled then
+// you need to have a permission with action `settings:write` with scope `settings:auth.<provider>:*`.
```

Comment:
I don't think we need this, because RBAC is enabled by default since v10.
```suggestion
// You need to have a permission with action `settings:write` with scope `settings:auth.<provider>:*`.
```